lib/common/memory/src/mmap_type.rs

```diff
@@ -365,17 +370,26 @@ impl MmapBitSlice {
             .truncate(true)
             .open(path)?;
 
-        let len = bitslice.len();
-        file.set_len(len as u64)?;
+        let bits_count = bitslice.len();
 
-        let mut mmap = unsafe { MmapMut::map_mut(&file)? };
-        mmap.fill(0);
+        let bytes_count = bits_count.div_ceil(u8::BITS as usize);
+
+        let bytes_count = max(Self::MIN_FILE_SIZE, bytes_count.next_power_of_two());
```

Comment:
Why `next_power_of_two`? Doesn't this mean we would have 50% overhead worst case?

Reply:
right, I think it should be `.next_multiple_of(size_of::<usize>())` instead.